deploy/kubernetes/base/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 commonLabels:
-  app: gcp-compute-persistent-disk-csi-driver
+  k8s-app: gcp-compute-persistent-disk-csi-driver
```

Comment:
add note to readme

Reply:
I don't think this needs to be documented - it's easy for the entry to be out of date with documentation, and users typically don't have to interact with this label